File: qc-external-pg-api/src/main/java/com/eager/questioncloud/pg/portone/PortonePaymentAmount.java
package com.eager.questioncloud.pg.portone;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Getter;

@Getter
@JsonIgnoreProperties(ignoreUnknown = true)
public class PortonePaymentAmount {
    private int total;
    private int taxFree;
    private int vat;
}

File: qc-external-pg-api/src/main/java/com/eager/questioncloud/pg/portone/PortonePaymentAmount.kt
package com.eager.questioncloud.pg.portone

import com.fasterxml.jackson.annotation.JsonIgnoreProperties

@JsonIgnoreProperties(ignoreUnknown = true)
class PortonePaymentAmount(
    val total: Int,
    val taxFree: Int,
    val vat: Int,
)
